packages/manager/modules/pci/src/projects/project/public-gateways/service/public-gateways.service.js

```diff
@@ -144,4 +169,18 @@ export default class publicGatewaysServiceClass {
       };
     });
   }
+
+  checkGatewayStatus(serviceName, operationId, namespace) {
```

Comment:
**nitpick** Since the function checks the status of gateway and private network creation, it is a generic function. So generic name would be better.
```suggestion
  checkOperationStatus(serviceName, operationId, namespace) {
```

Reply:
updated as suggested.